lib/proxy.js

```diff
@@ -1,20 +1,20 @@
 /*
-Copyright 2020 Adobe
+Copyright 2021 Adobe
```

Comment:
same remark as below:
@bartoszglow - why switch the copyright year? Shouldn't it stay as the year the file was first created? Same comment for the other files.